index.html

```diff
@@ -221,6 +221,30 @@ <h3>Skills</h3>
 
     </section>
 
+    <footer class="con-form-wrapper">
+      <div class="contact-form">
+        <h2 class="contact-me">Contact me</h2>
+        <p class="contact-description">If you have an application you are interested in developing, a feature that you
+          need built or a project that needs coding. I’d love to help with it.</p>
+        <form id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/mrgjgzbv"
+          method="post">
+          <div class="frm-inputs-cont">
+            <label for="full-name"></label>
+            <input class="con-frm-input" type="text" name="name" id="full-name" placeholder="First and Last Name"
+              required maxlength="30">
+            <label for="email-address"></label>
+            <input class="con-frm-input" type="email" name="_replyto" id="email-address"
+              placeholder="email@microverse.org" required>
+            <label for="message"></label>
+            <textarea class="con-frm-mess" rows="5" name="message" id="message" placeholder="Write your message here"
```

Comment:
- [ ] [OPTIONAL] Well done ✅ everything is working fine. I recommend to add some space here to improve your design and make it look like figma design 🍀😃

![image](https://user-images.githubusercontent.com/69011963/145217514-5ddacf4a-d5ae-4d1b-9291-ade95b836bef.png)